documentation/API proposals/APIproposal_QualityByDesign_CableLabs.md

```diff
@@ -0,0 +1,16 @@
+| API summary                               | API allowing 3rd party applications to:<br>- report appliction KPIs to network operator<br>- receive QbD score<br>- receive root cause and recommended corrective actions<br>- request service improvements                                                                                                                                                                                                                                                                                                                                                                                         |
+|                                           | **API Scope**<br>This API would be part of the Home Device API Family. The QbD API allows a 3rd party application developer to create a session with a network operator. The application can then share KPIs such as:<br>- latency<br>- jitter<br>- packet loss<br>- bitrate<br>The network operator can monitors these KPIs and determine a QbD score. If the score ever drop below certain thresholds, an event can be triggered for rapid telemetry collection. The network operator can then determine root cause and suggest corrective actions either to the 3rd party app or to the network. |
+|                                           | **Use Case Example 1 - Wi-Fi Congestion**<br>Wi-Fi congestion at a `Service Site` is affecting video conferencing performance, reflecting lower QbD scores and poor video quality                                                                                                                                                                                                                                                                                                                                                                                                                   |
+|                                           | **Use Case Example 2 - Network Impairments**<br>Issues in network operator's network can lead to degradation in latency, jitter, packet loss, and bitrate, reflecting lower QbD scores and poor video quality.                                                                                                                                                                                                                                                                                                                                                                                      |
```

Comment:
Instead of `network operator's network`, which is only used by Charter in their use cases or when spelling out MNO.  How about `access network` or `operator's network` or `service provider's` network.

Reply:
thanks - I like `operator's network` and went with that.